projects/element-ng/help-button/si-help-button.component.scss

```diff
@@ -0,0 +1,25 @@
+@use '@siemens/element-theme/src/styles/variables';
+@use 'sass:map';
+
+:host {
+  display: inline-flex;
+  vertical-align: middle;
+  align-items: center;
+  justify-content: center;
+  block-size: 1rem;
+  inline-size: 1rem;
+  font-size: 1.331rem;
```

Comment:
How did you end-up with this magic number? Ideally, we should use an established font-class, since this value actually doesn't adhere to the grid (`21.296px`). If you tried to compensate for the bounding box, ideally add a proper formula, so we can adjust it if we ever have to scale it.